data/api/build.gradle

```diff
@@ -10,7 +10,9 @@ apply plugin: 'kotlinx-serialization'
 apply from: rootProject.file('gradle/android.gradle')
 android.libraryVariants.all {
     // TODO replace with https://issuetracker.google.com/issues/72050365 once released.
-    it.generateBuildConfig.enabled = true
```

Comment:
Thanks!
Probably this gradle file load `gradle/android.gradle`. So can we delete this?

Reply:
gradle/android.gradle set generateBuildConfig = false.
But data/api subprojects use BuildConfig.java (to set API_ENDPOINT).
So this line seems necessary.

Reply:
Thanks. You are correct!! 👍